docs/extensions-plugins/chrome.mdx

```diff
@@ -4,7 +4,9 @@ description: Learn about the Pieces Chrome Extension, how it functions, and what
 ---
 
 <link rel="canonical" href="https://docs.pieces.app/extensions-plugins/chrome" />
-
+export const ChromeLogo = props  => {
+    return {{ marginTop: "0px", marginBottom: "1em", width: "50%" }} src={ "/assets/chrome_logo.png" }/>
```

Comment:
@CBID2 here you will actually need to include the `<img style=...` inside of the return on this ChromeLogo here, and it will look more like this: 

```
export const ChromeLogo = props  => {
    return <img style={{ marginTop: "0px", marginBottom: "1em", width: "50%" }} src={ "/assets/chrome_logo.png" }/>
}
```

there are some more docs on this here: 

https://use.docs.page/components/custom